### PR TITLE
chore(website): add TypeScript module system option for playground

### LINKED ISSUE #1173: Add module system selection in the playground
#### Reason/Context
For the TypeScript generator, you can [choose which module system the models should be generated for](https://github.com/asyncapi/modelina/blob/master/docs/languages/TypeScript.md#rendering-complete-models-to-a-specific-module-system), this should be possible in the playground as well.

To achieve this the following needs to change:
- Add the option `tsModuleSystem` to [the list of options](https://github.com/asyncapi/modelina/blob/c12c56ad705de89e1d0f80d0ec5706fe7b648139/modelina-website/src/types/index.ts#L12), where the type is `'ESM' | 'CJS' | undefined`
- Make sure we show the options being provided in [the generator code output of the playground](https://github.com/asyncapi/modelina/blob/c12c56ad705de89e1d0f80d0ec5706fe7b648139/modelina-website/src/helpers/GeneratorCode/TypeScriptGenerator.ts#L4).
- Adapt the UI to include a dropdown to [choose between the module systems](https://github.com/asyncapi/modelina/blob/c12c56ad705de89e1d0f80d0ec5706fe7b648139/modelina-website/src/components/playground/options/TypeScriptGeneratorOptions.tsx#L44)
- Add a react handler to handle when the [selected module system changes value](https://github.com/asyncapi/modelina/blob/c12c56ad705de89e1d0f80d0ec5706fe7b648139/modelina-website/src/components/playground/options/TypeScriptGeneratorOptions.tsx#L32)
- Adapt the backend serverless function to use the [new option during the generation phase](https://github.com/asyncapi/modelina/blob/c12c56ad705de89e1d0f80d0ec5706fe7b648139/modelina-website/src/pages/api/functions/TypeScriptGenerator.ts#L21)

Happy to help even further in case you need it! Ping me here in the issue or in the AsyncAPI slack channel 👋

### PR DESCRIPTION
**Pull request by ishaan812**
**Description**
- Added option for generation of TypeScript using different module systems
- Added UI to support this operation in Playground

image

**Related issue(s)**
Resolves #1173

### CODE REVIEW

File: modelina-website/src/components/playground/options/TypeScriptGeneratorOptions.tsx [hunk 1]
```diff
@@ -44,6 +45,13 @@ class TypeScriptGeneratorOptions extends React.Component<
     }
   }
 
+  onChangeModuleSystem(moduleSystem: any) {
+    if (this.props.setNewConfig) {
+      this.props.setNewConfig('tsModuleSystem', String(moduleSystem));
+      console.log(moduleSystem)
```
Comment (jonaslagoni) [inline modelina-website/src/components/playground/options/TypeScriptGeneratorOptions.tsx:51]: This needs to be removed 🙂

**Review by jonaslagoni — CHANGES_REQUESTED** [review]
Just one change 😄

**Comment by ishaan812** [comment]
Yup should be good now

**Review by ishaan812 — COMMENTED** [review]
made the change

**Review by jonaslagoni — APPROVED** [review]
LGTM 👍

**Comment by jonaslagoni** [comment]
@all-contributors please add @ishaan812 for code

**Pull request merged**